safe/common/test/test_version.py

```diff
@@ -58,8 +58,9 @@ def test_get_version(self):
     def test_get_current_hash(self):
         """Test for get_current_hash."""
         git_hash = current_git_hash()
-        if 'win32' in sys.platform:
+        if 'win32' in sys.platform or 'darwin' in sys.platform:
```

Comment:
The output of is `win32`: http://stackoverflow.com/questions/446209/possible-values-from-sys-platform
ok